### ISSUE TITLE
Feature Request: Self serve client files

### ISSUE THREAD

**Issue by haxwithaxe** [RobinLinus/snapdrop#614]
This is mostly to gauge interest in a potential pull request.

It would be nice to have an option for the server to host the client files itself. This is adding complexity but only if you only count the nodejs code. As a system this will at worst shift complexity from one part to another and probably effectively reduce complexity overall. It would simplify deployment. If you are interested in this but don't want to implement it I can try it and send a pull request. If you're not interested then you can just close this.

Thanks for maintaining this project :)

**Comment by stemid** [comment]
Yes, I just found this project and this is a major gripe with deployment. That I need two containers just to serve this application. Which is very rare, I host a lot of things for myself and my friends.

I'm in fact right now making my own container image with darkhttpd, just because I feel that nginx is overkill to serve some static files. And I don't want any TLS generation because I use a reverse proxy with ACME.

**Comment by schlagmichdoch** [comment]
> If you are interested in this but don't want to implement it I can try it and send a pull request.

I do not post in this repo normally in order to not annoy the creator. I do now only to prevent you from wasting your work and time as this is already implemented but he does not seem to be interested in PRs to be merged.

Please take a look at https://github.com/schlagmichdoch/PairDrop and create an issue over there if you want to answer to this comment.
